MetaMask/metamask-mobile
# feat: update text of swaps gas tooltip

## **Description**

Updates the swap gas tooltip text

## **Manual testing steps**

1. Enter info for a swap
2. Proceed to the review screen
3. Click on the tooltip
4. Confirm that the tooltip has been updated to say, "MetaMask doesn't make money from gas fees. These fees are estimates and can change based on how busy the network is and how complex a transaction is. Learn more here."  [“here” is a hyperlinkl that should link to help center FAQs on gas fees: [Gas fees | MetaMask Help Center 🦊♥️](https://support.metamask.io/transactions-and-gas/gas-fees/) ]

## **Screenshots/Recordings**

### **Before**

<img width="296" alt="Screenshot 2024-09-17 at 9 10 45 AM" src="https://github.com/user-attachments/assets/001577cd-90f9-4ad8-a3ff-64f4458a8360">



### **After**

<img width="287" alt="Screenshot 2024-09-17 at 1 42 11 PM" src="https://github.com/user-attachments/assets/365b93ba-6965-4723-9e9f-c14aa90bdc93">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
